spec/mongoid/contextual/mongo_spec.rb

```diff
@@ -3292,6 +3292,12 @@
     it "limits the results" do
       expect(context.skip(1).entries).to eq([ new_order ])
     end
+
+    context "with #last" do
+      it "returns the nth from last element" do
+        expect(context.skip(1).last).to eq(depeche_mode)
+      end
+    end
```

Comment:
The method has `sort(inverse_sorting)` and then later `v.to_a.reverse`, which ensure the ordering in the case hat multiple docs are returned.

Are there cases for the following, which ensure correct document ordering the result for multiple docs?
- combining `skip(n)` with `limit(m)` (where n, m > 1)
- `last(n)` (where n > 1)